dateparser_data/supplementary_language_data/date_translation_data/en.yaml

```diff
@@ -48,3 +48,18 @@ simplifications:
     - (\d+)h(\d+)m?: \1:\2
     - (?<=from\s+)now: in
     - less than 1 minute ago: 45 second ago
```

Comment:
I see all of this makes sense, but exactly for that I don't understand why wasn't this add before. @ArturGaspar would you like give your input here please?

Reply:
Because of precision. It may well enough be `1 second ago`, `30 seconds ago` or `59.99 seconds ago`. I think we need a setting for that, because `45 seconds` is a size that will not fit all.

Previously discussed under: https://github.com/scrapinghub/dateparser/issues/411#issuecomment-387643982

Reply:
You seem to be discussing a line that is not part of these changes here.

Reply:
I think I was talking about the numbers or I don't remember, so I'll be closing this conversation threa.